javascript/modules/client/src/context.ts

```diff
@@ -5,7 +5,7 @@ import { UnsupportedProtocolError } from "@aragon/sdk-common";
 // import { GraphQLClient } from "graphql-request";
 export { ContextParams } from "./internal/interfaces/context"
 
-const supportedProtocols = ["https:"];
+const supportedProtocols = ["http:", "https:"];
```

Comment:
@marcvelmer if "http" is need only for the testing then i would try to have it overriden from the testing files

Reply:
+1